llvm_wrapper.py

```diff
@@ -6,17 +6,26 @@
 
 class CConfig(object):
     _includes = "llvm-c/Core.h llvm-c/BitReader.h"
-    output, error = Popen("llvm-config --libs core native --system-libs", 
-                    shell=True, stdout=PIPE, stderr=PIPE).communicate()
+    libs = Popen("llvm-config --libs core native --system-libs", 
```

Comment:
This is nitpicking, but usually when you break a line inside parenthesis, you line up the next line with the open parenthesis. E.g.:

```
foo(this_is_a_long_line, with_many_arguments, lalala, lalala,
    next_line_aligned_like_this)
```